ui/de-webapp/src/main/java/org/iplantc/de/server/controllers/file/DownloadController.java

```diff
@@ -0,0 +1,147 @@
+package org.iplantc.de.server.controllers.file;
+
+import static org.iplantc.de.server.AppLoggerConstants.REQUEST_KEY;
+import static org.iplantc.de.server.AppLoggerConstants.RESPONSE_KEY;
+
+import org.iplantc.de.server.AppLoggerConstants;
+import org.iplantc.de.server.AppLoggerUtil;
+import org.iplantc.de.server.auth.DESecurityConstants;
+import org.iplantc.de.server.auth.JwtBuilder;
+import org.iplantc.de.shared.services.BaseServiceCallWrapper;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.jose4j.lang.JoseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created by sriram on 6/28/16.
+ */
+public class DownloadController {
+
+    final Logger API_REQUEST_LOG = LoggerFactory.getLogger(AppLoggerConstants.API_METRICS_LOGGER);
+    final AppLoggerUtil loggerUtil = AppLoggerUtil.getInstance();
+    final AppLoggerUtil appLoggerUtil = loggerUtil;
```

Comment:
I had the same question. I will check with @strootman and make changes if necessary!